pmoves/services/hi-rag-gateway-v2/embeddings.py

```
@@ -0,0 +1,62 @@
+    if _embedder is None:
+        from sentence_transformers import SentenceTransformer
+        _embedder = SentenceTransformer(MODEL, device=DEVICE)
+    return _embedder
+
+
+def embed_query(text: str):
+    # Try provider chain first; fall back to local SentenceTransformer when unavailable
+    vec = None
+    try:
+        vec = _embed_via_providers(text)
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)